### PR TITLE
made a module out of `hmr` to make it easier to move between projects

### PR DESCRIPTION
**Pull request by basarat**
:rose:

With docs https://www.npmjs.com/package/fuse-hmr :memo:

### CODE REVIEW

File: src/client/application.tsx [hunk 1]
```diff
@@ -30,5 +30,6 @@ export const render = (element) => {
         document.querySelector(element)
     );
 }
-import { setStatefulModules } from './hmr';
-setStatefulModules('hmr', 'store/', 'actions/');
+import { setStatefulModules } from 'fuse-hmr';
+declare var FuseBox;
+setStatefulModules(FuseBox, ['hmr', 'store/', 'actions/']);
```
Comment (basarat) [inline src/client/application.tsx:35]: @nchanged FuseBox loader plugins shared via NPM will need to take `FuseBox` from the user code base. This is the first package sharing FuseBox loader plugins so more patterns will emerge, but worthwhile for you to know :rose:

**Comment by nchanged** [comment]
Thanks!
Can you move it here please? https://github.com/fuse-box/fuse-hmr

**Comment by basarat** [comment]
Will do :rose: :heart:

**Comment by basarat** [comment]
That will fix `FuseBox` global issue as well :)

**Pull request closed without merging**